src/test/lrsql/admin/protocol_test.clj

```diff
@@ -155,6 +183,49 @@
         (is (-> (adp/-ensure-account-oidc lrs username bad-issuer)
                 :result
                 (= :lrsql.admin/oidc-issuer-mismatch-error)))))
+    (testing "actor-delete"
+      (let [stmts [stmt-0 stmt-1 stmt-2 stmt-3]
+            ifis (->> (conj stmts (stmt-3 "object"))
+                      (map #(ua/actor->ifi (% "actor"))))
+            get-stmt-#s (fn []
+                          (reduce (fn [m actor-ifi]
+                                    (assoc m actor-ifi
+                                           (count (:statements (:statement-result (lrsp/-get-statements lrs auth-ident {:actor-ifi actor-ifi} []))))))
+                                  {} ifis))]
+        (lrsp/-store-statements lrs auth-ident stmts [])
+
+        (assert (every? #(> % 0) (vals (get-stmt-#s))))
+        (doseq [ifi ifis]
+          (adp/-delete-actor lrs {:actor-ifi ifi}))
+        (is (every? #(= % 0) (vals (get-stmt-#s))))))
+    (let [stmt->ifi #(ua/actor->ifi (% "actor"))
+          count-of-actor (fn [actor-ifi] (-> (lrsp/-get-statements lrs auth-ident {:actor-ifi actor-ifi} []) :statement-result :statements count))
+          child-ifi (stmt->ifi (stmt-3 "object"))
+          parent-ifi (stmt->ifi stmt-3)]
```

Comment:
I think you can put these `let` bindings in the outer `let`. That way you can get rid of one level of nesting.